src/pages/index.js

```diff
@@ -15,64 +17,177 @@ const cardFormElement = document.querySelector("#add-card-form");
 //Buttons and other DOM nodes
 const profileEditButton = document.querySelector(".profile__edit-button");
 const newCardButton = document.querySelector(".profile__add-button");
+const confirmButton = document.querySelector("#confirm-modal");
 
 //Form data
 const nameInput = profileEditModal.querySelector("#profile-title");
 const jobInput = profileEditModal.querySelector("#description");
 
-//profile form
-const userInfo = new UserInfo(".profile__title", ".profile__description");
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "9a1167a0-01e6-4e49-b3c5-e813c03c355c",
+    "Content-Type": "application/json",
+  },
+});
+
+//information from server
+
+let cardList;
+let userInfo;
+
+api
+  .loadPageResults()
+  .then(([cards, userData]) => {
+    console.log(cards);
+    console.log(userData);
+    cardList = new Section(
+      {
+        items: cards,
+        renderer: createCard,
+      },
+      ".gallery__cards"
+    );
+    cardList.renderItems();
+    userInfo = new UserInfo(
+      ".profile__title",
+      ".profile__description",
+      ".profile__image"
+    );
+    userInfo.setUserInfo(userData);
+  })
+  .catch((err) => console.error(err));
+
+//Render Loading
+function renderLoading(isLoading) {
+  if (isLoading) {
+  }
+}
+
+//edit profile information
 
 function handleProfileFormSubmit(userData) {
-  profilePopupForm.close();
-  userInfo.setUserInfo(userData);
+  profilePopupForm.setLoading(true, "Saving...");
+  api
+    .editProfile(userData)
+    .then((res) => {
+      profilePopupForm.close();
+      api.getInfo();
+      return res;
+    })
+    .then((res) => userInfo.setUserInfo(res))
+    .catch((err) => console.error(err))
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors